data/events/scripts/player.lua

```diff
@@ -725,14 +725,8 @@ function Player:onGainExperience(target, exp, rawExp)
 	end
 
 	local baseRate = self:getFinalBaseRateExperience()
-	local finalExperience
-	if configManager.getBoolean(configKeys.RATE_USE_STAGES) then
-		finalExperience = (exp * baseRate + (exp * (storeXpBoostAmount/100))) * staminaBoost
-	else
-		finalExperience = (exp + (exp * (storeXpBoostAmount/100))) * staminaBoost
-	end
 
-	return math.max(finalExperience)
+	return math.max((exp * baseRate + (exp * (storeXpBoostAmount/100))) * staminaBoost)
```

Comment:
Why even call math.max? You are basically saying math.max(singleNumber), it will always just return whatever the number was, you can just remove the call to it if it is only comparing one number.